auctions/README.md

```diff
@@ -0,0 +1,43 @@
+# Auctions
+
+## Firestore collections:
+
+- auctions
+- biddingLog
+
+## Auctions
+```
+{
+  'id': <auction-id>,
+  'seller': <userId>,
+  'item_type': <example: neelam>,
+  'quantity': <number>,
+  'highest_bidder': <userId>,
+  'highest_bid': <example: 200 dinero | default: initial_price>,
+  'start_time': <unix-timestamp>,
+  'end_time': <unix-timestamp>,
+  'bidders': [<userId>, <userId>]
+}
+```
+
+## biddingLog
+```
+{
+  'id': <auction-id>,
+  'seller': <userId>,
+  'item_type': <currency: neelam>,
+  'quantity': <number: 10>,
+  'initial_price': <example: 100 dinero>,
+  'highest_bidder': <userId>,
+  'highest_bid': <example: 200 dinero | default: initial_price>,
+  'start_time': <unix-timestamp>,
+  'end_time': <unix-timestamp>,
+  'bidders_and_bids': [
+    {
+      'bidder': <userId>,
+      'bid': <number>,
+      'timestamp': <unix-timestamp>
+    }
+  ],
+}
+```
```

Comment:
This is how we're returning data and not how we're storing it in firestore

This is how we're storing data in `auctions` collection:
![image](https://user-images.githubusercontent.com/56217868/116403580-5bbc1b00-a84b-11eb-9d54-41f49f20ff21.png)

`bids` collection looks like this:
![image](https://user-images.githubusercontent.com/56217868/116403604-65458300-a84b-11eb-9ef5-c00c9a2ad0e4.png)

Reply:
Updated structure based on screen shot above.